exchange/bidder.go

```diff
@@ -58,17 +58,32 @@ type AdaptedBidder interface {
 	//
 	// Any errors will be user-facing in the API.
 	// Error messages should help publishers understand what might account for "bad" bids.
-	requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, bidRequestOptions bidRequestOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, hookExecutor hookexecution.StageExecutor, ruleToAdjustments openrtb_ext.AdjustmentsByDealID) ([]*entities.PbsOrtbSeatBid, []error)
+	requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, requestBidInfo *extraRequestBidInfo, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, hookExecutor hookexecution.StageExecutor, ruleToAdjustments openrtb_ext.AdjustmentsByDealID) ([]*entities.PbsOrtbSeatBid, []error)
```

Comment:
Why is `requestBidInfo` a pointer?

Reply:
```
type extraRequestBidInfo struct {
	request  bidderRequestInfo
	response *bidderResponseInfo
}
```

`response` from the `requestBidInfo` is pointer. so we don't need entire `requestBidInfo` to be pointer.  @hhhjort https://github.com/prebid/prebid-server/pull/2874/commits/f6ba79f617b394ae3e226f7d0e156dc40839ffc9 makes this change. Thanks